app/bundles/CoreBundle/IpLookup/AbstractMaxmindLookup.php

```diff
@@ -65,8 +65,12 @@ protected function parseResponse($response)
                 if (isset($data->postal)) {
                     $this->zipcode = $data->postal->code;
                 }
-                $this->country = $data->country->names->en;
-                $this->city    = $data->city->names->en;
+                if (isset($data->country)) {
+                    $this->country = $data->country->names->en;
+                }
+                if (isset($data->city)) {
+                    $this->city    = $data->city->names->en;
+                }
```

Comment:
Nitpick: It could be written more elegantly with PHP 7 syntax:
```php
$this->country = $data->country->names->en ?? null;
$this->city    = $data->city->names->en ?? null;
```

Reply:
@escopecz I'm more than happy to "upgrade" the syntax, but I figured if I followed what was already happening I had the highest chance of getting the PR through.

I could follow up with another PR to pretty up the syntax if you'd like?

Reply:
No need. I understand the reasoning and this will work just fine. Thanks!